### ISSUE TITLE
 add WOL for ethernet devices and team/bond slaves

### ISSUE THREAD

**Issue by vbenes** [cockpit-project/cockpit#4390]
we need to implement wake-on-lan functionality as supported in NM and GNOME control-center for ethernet devices and slaves.

This can be seen in nm-connection-editor (and control-center too)

**Comment by vbenes** [comment]
wake on lan password seems to be important as well

**Comment by vbenes** [comment]
bridge slaves should have been included too

**Comment by stefwalter** [comment]
Is this ever used on servers? What's the full use case here?

**Comment by vbenes** [comment]
actually this is supported by nm-connection-editor and GNOME control-center and nmcli/nmtui and is quite used by customer so probably worth implementing

see https://bugzilla.redhat.com/show_bug.cgi?id=1270194 for example

**Comment by mvollmer** [comment]
> https://bugzilla.redhat.com/show_bug.cgi?id=1270194 

I am unable to access that page.  Can you summarize the use case?

I can't find the setting in GNOME Control Center, only in nm-connection-editor.  However, nm-connection-editor doesn't let me make me make changes to the WoL radio buttons.

**Comment by marusak** [comment]
This issue is closed due to inactivity.

We are sorry, but we won't be able to address this in any time soon.
If you are interested in helping us with the implementation, please comment here.
